Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -976,6 +978,10 @@ internal struct LongPathInfo {
                     _analyzer = null;
                 }
 
+                if (_idleNodeModulesTimer != null) {
+                    _idleNodeModulesTimer.Dispose();

```

Comment:
The following event listeners on fileWatcher are subscribed to, but never unsubscribed from - these trigger the timer. 
```cs
    _nodeModulesWatcher.Changed += OnNodeModulesWatcherChanged;
    _nodeModulesWatcher.Created += OnNodeModulesWatcherChanged;
    _nodeModulesWatcher.Deleted += OnNodeModulesWatcherChanged;
```